test/IRGen/jit_metadata_strategy.swift

```diff
@@ -0,0 +1,39 @@
+// RUN: %target-swift-frontend -interpret %s
+// REQUIRES: executable_test
```

Comment:
I think that this is better tested in terms of IRGen, rather than checking that the executable executes.

Reply:
You could also have both.

Reply:
Sure, as long as they are in separate files.  The reason for preferring checking IRGen is that means that you can verify IRGen irrespective of the target that you are on.  This massively helped for example in the Windows bring up where I did the entirety of the work on Linux before I started running the executable tests.